data/json/items/comestibles/raw_veggy.json

```
@@ -91,7 +92,6 @@
     "calories": 33,
     "spoils_in": "20 days",
     "description": "A slightly sweet tuber.  Contains much inulin which may cause some minor issues with your digestive tract, so it should be cooked.",
-    "comestible_type": "FOOD",
```

```suggestion
    "comestible_type": "FOOD",
```
Moving this around is unecessary